terraform/testnet/testnet/templates/monitoring.yaml

```diff
@@ -243,6 +243,8 @@ spec:
         env:
         - name: GF_AUTH_SIGV4_AUTH_ENABLED
           value: "true"
+        - name: AWS_SDK_LOAD_CONFIG
+          value: "true"
```

Comment:
what happen when there's no config? can you set those variable to false by default?

Reply:
It's used in conjunction with the EKS IAM role annotation to grant Grafana the permission to sign requests.

Reply:
Changed it actually so that we can pass in all the envs at once using helm value. This way it's more clear and has a default:

```
monitoring:
  grafana:
    env:
      GF_AUTH_SIGV4_AUTH_ENABLED: true
      AWS_SDK_LOAD_CONFIG: true
```